### DISCUSSION TITLE
Unable to change Icon color

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#3887] [Q&A] [answered]
Hi,

I am unable to assign custom color property to Icon.

`<Icon color="green.400" as={GrFormNext} boxSize={10} />`

Sandbox [link](https://codesandbox.io/s/hungry-dew-3ijvh?file=/src/index.js).

:european_castle:

**Comment by chr-ge** [comment, marked as answer]
It looks like those Grommet-Icons have a set stroke color. If you inspect the icon you will see `stroke="#000"` set on the `path`. This is unlike other icons that have no set stroke color and use `stroke="currentColor"` on the `svg` element.

- You can either copy the icon's svg and paste it into your component and remove that stroke attribute or 
- use another icon pack.

**Reply by spidertwin2** [reply]
thanks @chr-ge..I switched to `IoIosArrowForward`.

:pray:
